src/audio/module_adapter/iadk/system_agent.cpp

```diff
@@ -14,7 +14,7 @@
 #include <stdint.h>
 #include <rtos/string.h>
 #include <utilities/array.h>
-#include <adsp_error_code.h>
```

Comment:
so now we have sof_api and iadk ?

Should not IADK go to sof_api as well?

Reply:
what is the difference?